src/azure-cli-core/azure/cli/core/config_defaults.py

```diff
@@ -0,0 +1,264 @@
+                    "description": ""
+                },
+                {
+                    "value": "none",
+                    "description": ""
+                }
+            ],
+            "type": "string",
+            "description": "Color theme for terminal."
+        },
+        "cache_ttl": {
+            "default": "10",
+            "allowed": [],
+            "type": "integer",
+            "description": "How long (in minutes) to before invalidating cache"
+        }
+    },
+    "cloud": {
+        "name": {
+            "default": "AzureCloud",
+            "allowed": [
+                {
+                    "value": "AzureCloud",
+                    "description": ""
+                },
+                {
+                    "value": "AzureChinaCloud",
+                    "description": ""
+                },
+                {
+                    "value": "AzureUSGovernment",
+                    "description": ""
+                },
+                {
+                    "value": "AzureGermanCloud",
+                    "description": ""
+                },
+                {
+                    "value": "Other registered cloud - More information about registering new cloud: https://docs.microsoft.com/en-us/cli/azure/manage-clouds-azure-cli#register-a-new-cloud",
+                    "description": ""
+                }
+            ],
+            "type": "string",
+            "description": "The default cloud for all az commands"
+        }
+    },
+    "query": {
+        "allow_list": {
+            "default": "list, show",
+            "allowed": [
+                {
+                    "value": "list",
+                    "description": ""
+                },
+                {
+                    "value": "show",
+                    "description": ""
+                },
+                {
+                    "value": "version",
+                    "description": ""
+                },
+                {
+                    "value": "'' (leave blank)",
+                    "description": "Do not generate query examples."
+                }
+            ],
+            "type": "Comma separated values",
+            "description": "Generate JMESPath query examples for commands ending with string in this list."
+        },
+        "examples_len": {
+            "default": "80",
+            "allowed": [],
+            "type": "integer",
+            "description": "Maximum length for example query string"
+        },
+        "help_len": {
+            "default": "80",
+            "allowed": [],
+            "type": "integer",
+            "description": "Maximum length for example help string"
+        },
+        "max_exmples": {
+            "default": "80",
+            "allowed": [],
+            "type": "integer",
+            "description": "Maximum number of examples to generate"
+        }
+    },
+    "extension": {
+        "use_dynamic_install": {
+            "default": "no",
```

Comment:
The default value has been changed to `yes_prompt`.

Reply:
fixed in e21cdea